.github/workflows/docker.yml

```diff
@@ -0,0 +1,40 @@
+name: Build-Env Docker Image
+
+on:
+  push:
+    branches: master
```

Comment:
This will build and push the image when changes are made to the two files under the 'paths' key on the master branch. It won't do anything for pull requests.